src/Adapter/Product/QueryHandler/GetProductForEditingHandler.php

```diff
@@ -170,15 +170,11 @@ private function getCategoriesInformation(Product $product): ProductCategoriesIn
     private function getPricesInformation(Product $product): ProductPricesInformation
     {
         $priceTaxExcluded = $this->numberExtractor->extract($product, 'price');
-        $taxRulesGroup = $this->taxRulesGroupRepository->getTaxRulesGroupDetails(new TaxRulesGroupId((int) $product->id_tax_rules_group));
-        if (!empty($taxRulesGroup['rates'])) {
-            // Use the tax rate associated to context country, or the first one as fallback
-            $countryTaxRate = $taxRulesGroup['rates'][$this->countryId] ?? reset($taxRulesGroup['rates']);
-        } else {
-            $countryTaxRate = 0;
-        }
-        $taxRatio = new DecimalNumber((string) (1 + ($countryTaxRate / 100)));
-        $priceTaxIncluded = $priceTaxExcluded->times($taxRatio);
+        $priceTaxIncluded = $this->taxComputer->computePriceWithTaxes(
```

Comment:
Where is the calculation about ecotax ?

Reply:
I never worked with ecotaxes yet, but isn't it fetched from tax rule groups like other taxes? (`tax_rules_group` in DB)

Reply:
But the ecotax isn't a tax

Reply:
you mean because it increases the "out of tax" value?
well I'll wait for @jolelievre 's answer, I don't master this subject 👍

Reply:
The ecotax is not handled yet, I'm not sure we'll have time to handle it before freeze as it's not in Must Have, but I'll try now that the JS model is merged
Anyway it's not in the scope of this PR ^^

Reply:
But i don"t see any issue for the ecotax